packages/alfa-rules/src/sia-r8/rule.ts

```diff
@@ -1,6 +1,6 @@
-import { Rule } from "@siteimprove/alfa-act";
+import { Diagnostic, Rule } from "@siteimprove/alfa-act";
 import type { Role } from "@siteimprove/alfa-aria";
-import { DOM } from "@siteimprove/alfa-aria";
+import { DOM, Node as AlfaAriaNode } from "@siteimprove/alfa-aria";
```

Comment:
**Nitpick:** in other places, we usually `import * as aria from "@siteimprove/alfa-aria"`  and then use `aria.Node`.